### PR TITLE
Migrating from sphinx-panels to sphinx-design

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

:rotating_light: sphinx-panels is not actively maintained. Use [sphinx-design](https://github.com/executablebooks/sphinx-design) instead! See [the migration guide](https://sphinx-design.readthedocs.io/en/latest/get_started.html#migrating-from-sphinx-panels) and https://github.com/executablebooks/sphinx-design/issues/51 for more information. :rotating_light:

 

### Details

- [x] Compare the main and this PR's sphinx document.

### CODE REVIEW

File: doc/api/index.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ Highlights of the API include:
    readers/index
    examples/index
 
-.. panels::
+.. grid:: 1 2 2 2
```
Comment (tkoyama010) [inline doc/api/index.rst:24]:
```suggestion
.. grid:: 1 2 2 2
    :gutter: 2
```

File: doc/getting-started/index.rst [hunk 2]
```diff
@@ -11,7 +11,7 @@ Getting Started
    external_examples
 
 
-.. panels::
+.. grid:: 1 2 2 2
```
Comment (tkoyama010) [inline doc/getting-started/index.rst:14]:
```suggestion
.. grid:: 1 2 2 2
    :gutter: 2
```

File: doc/getting-started/index.rst [hunk 3]
```diff
@@ -55,7 +55,7 @@ Getting Started
 Installation
 ============
 
-.. panels::
+.. grid:: 1 2 2 2
```
Comment (tkoyama010) [inline doc/getting-started/index.rst:58]:
```suggestion
.. grid:: 1 2 2 2
    :gutter: 2
```

File: doc/getting-started/index.rst [hunk 4]
```diff
@@ -11,7 +11,8 @@ Getting Started
    external_examples
 
 
-.. panels::
+.. grid:: 1 2 2 2
+    :gutter: 2
     :column: col-lg-6 col-md-6 col-sm-12 col-xs-12 p-3
```
Comment (tkoyama010) [inline doc/getting-started/index.rst:16]:
```suggestion
```

File: doc/getting-started/index.rst [hunk 5]
```diff
@@ -55,7 +56,8 @@ Getting Started
 Installation
 ============
 
-.. panels::
+.. grid:: 1 2 2 2
+    :gutter: 2
     :card: + install-card
     :column: col-lg-6 col-md-6 col-sm-12 col-xs-12 p-3
```
Comment (tkoyama010) [inline doc/getting-started/index.rst:62]:
```suggestion
```

File: doc/api/index.rst [hunk 6]
```diff
@@ -21,15 +21,15 @@ Highlights of the API include:
...
+    :gutter: 2
 
     Core API
     ^^^^^^^^
 
     Learn more anout PyVista's different mesh types and direct access to common VTK filters.
 
-    .. link-button:: core/index
+    .. button-ref:: core/index
         :type: ref
```
Comment (tkoyama010) [inline doc/api/index.rst:33]:
```suggestion
```

Comment (tkoyama010) [inline doc/api/index.rst:33]:
```suggestion
```

File: doc/api/index.rst [hunk 7]
```diff
@@ -41,7 +41,7 @@ Highlights of the API include:
 
     Explore PyVista's robust plotting interface for visualizing the core data structures.
 
-    .. link-button:: plotting/index
+    .. button-ref:: plotting/index
         :type: ref
```
Comment (tkoyama010) [inline doc/api/index.rst:45]:
```suggestion
```

File: doc/api/index.rst [hunk 8]
```diff
@@ -53,7 +53,7 @@ Highlights of the API include:
 
     Use PyVista's Reader classes to read data files.
 
-    .. link-button:: readers/index
+    .. button-ref:: readers/index
         :type: ref
```
Comment (tkoyama010) [inline doc/api/index.rst:57]:
```suggestion
```

File: doc/api/index.rst [hunk 9]
```diff
@@ -65,7 +65,7 @@ Highlights of the API include:
 
     Utilize PyVista's helper modules, conversion tools, and geometric object creation routines.
 
-    .. link-button:: utilities/index
+    .. button-ref:: utilities/index
         :type: ref
```
Comment (tkoyama010) [inline doc/api/index.rst:69]:
```suggestion
```

**Comment by tkoyama010** [comment]
Thanks. I am searching for how I can convert sphinx-panels to sphinx-design. But it is complicated.

**Comment by akaszynski** [comment]
Doesn't look half bad:

tmp0

tmp1

**Comment by tkoyama010** [comment]
@akaszynski Thanks for improving this PR.

**Review by akaszynski — APPROVED** [review]
This is good to go and needed for the next release.

Weirdness in the side panel will be fixed by a follow-up PR. Root cause is `sidebar-nav-bs.html`.

**Review by banesullivan — APPROVED** [review]
Merge away

**Pull request merged**